src/Lykke.Service.Assets.Repositories/RepositoriesModule.cs

```diff
@@ -29,6 +29,8 @@ protected override void Load(ContainerBuilder builder)
             const string dictionaryTableName  = "Dictionaries";
             const string erc20TokenTableName  = "Erc20Tokens";
             const string watchListTableName   = "WatchLists";
+            const string assetConditionTableName = "AssetCondition";
```

Comment:
Table name not in plural form

Reply:
This name is used in production environment.